### PR TITLE
add antibodies in imaging path

### PR DESCRIPTION
**Pull request by andreacosolo**
- added antibodies to imaging_path as linkTo Antibody items
- in `display_title`, primary Ab and secondary Ab are equivalent to other_probes and labeled_probe, respectively
- display_title removes a label if already present in labeled probe
- display_title can be overridden
- embedded all fields from linkTo items used in their display titles
- since the ImagingPath display_title changed, modified the embedding list of all items that linkTo ImagingPath

### CODE REVIEW

File: src/encoded/types/imaging.py [hunk 1]
```diff
@@ -40,22 +47,41 @@ def display_title(self, request, target=None, labeled_probe=None, other_probes=N
...
+        if labeled_probes:
+            labeled_probes = ", ".join(labeled_probes)
+
+        primary_probes = []
         if other_probes:
-            other_probes = ", ".join(other_probes)
+            primary_probes.extend(other_probes)
+        if primary_antibodies:
+            for ab in primary_antibodies:
+                antibody = request.embed(ab, '@@object').get('antibody_name')
```
Comment (aschroed) [inline src/encoded/types/imaging.py:64]: another use of request.embed

File: src/encoded/types/imaging.py [hunk 2]
```diff
@@ -40,22 +47,41 @@ def display_title(self, request, target=None, labeled_probe=None, other_probes=N
                 target_title = request.embed(a_target, '@@object').get('display_title')
```
Comment (aschroed) [inline src/encoded/types/imaging.py:47]: Can you change from using request.embed  to get_item_or_none here (which I guess was pre-existing) and the following 2 instances - while they are really essentially the same the preferred usage is get_item_or_none - mainly I think due to better error handling?

**Review by aschroed — APPROVED** [review]
A couple of non blocking comments.  I presume you included enough test cases to ensure yourself that most if not all potential edge cases will be handled correctly.  Look good to me!

File: src/encoded/types/imaging.py (hunk 2 above)
Comment (andreacosolo) [reply src/encoded/types/imaging.py]: I just used what was there before, didn't know that this was the preferred usage. Changed now.

**Review by aschroed — APPROVED** [review]
This looks good to me.  You may want Will to have a quick look as the helper functions and dependencies to create the embedded lists are new but they look good to me.

**Review by willronchetti — APPROVED** [review]
